crates/bitcoind_rpc/src/lib.rs

```diff
@@ -144,6 +154,37 @@ impl<'c, C: bitcoincore_rpc::RpcApi> Emitter<'c, C> {
     }
 }
 
+/// A new emission from mempool.
+#[derive(Debug)]
+pub struct MempoolEvent {
+    /// Emitted mempool transactions with their first‐seen unix timestamps.
+    pub emitted_txs: Vec<(Transaction, u64)>,
+
+    /// Set of all [`Txid`]s from the raw mempool result, including transactions that may have been
+    /// confirmed or evicted during processing. This is used to determine which expected
+    /// transactions are missing.
+    pub raw_mempool_txids: HashSet<Txid>,
```

Comment:
```suggestion
    /// Txids of all transactions currently present in the mempool.
    ///
    /// This is useful to determine whether transactions are evicted from the mempool.
    pub all_txids: HashSet<Txid>,
```

I think this is more concise and clearer.